src/server/gc_worker/config.rs

```diff
@@ -27,6 +36,22 @@ pub struct GcConfig {
     pub compaction_filter_skip_version_check: bool,
     /// gc threads count
     pub num_threads: usize,
+
+    // Auto compaction settings
+    /// How often to check for new compaction candidates (in seconds)
+    pub auto_compaction_check_interval_secs: u64,
+
+    // Compaction threshold settings
```

Comment:
How about using an individual struct to encapsulate all relevant settings on auto compaction settings, for example:
```suggestion
// Compaction threshold settings
struct CompactionThreshold {
    /// Frequency to check for new compaction candidates (in seconds)
    pub check_interval_secs: u64,
    /// Minimum number of tombstones to trigger compaction
    pub tombstones_num: u64,
    /// Minimum percentage of tombstones to trigger compaction
    pub tombstones_percent: u64,
    /// Minimum number of redundant rows to trigger compaction
    pub redundant_rows: u64,
    /// Minimum percentage of redundant rows to trigger compaction
    pub redundant_rows_percent: u64,
    /// Force compaction of bottommost level
    pub bottommost_level_force: bool,
}    
```

Reply:
Done